third_party/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation - All rights reserved
+# Licensed under the MIT License
+
+
+SB_MICRO_PATH="${SB_MICRO_PATH:-/usr/local}"
+
+# Build cutlass.
+if [ -f cutlass/CMakeLists.txt ]; then
+    SOURCE_DIR=./cutlass
+    BUILD_ROOT=./cutlass/build
+    mkdir -p $BUILD_ROOT
+    cmake -DCMAKE_INSTALL_BINDIR=$SB_MICRO_PATH/bin -DCMAKE_INSTALL_LIBDIR=$SB_MICRO_PATH/lib -DCMAKE_BUILD_TYPE=Release -S $SOURCE_DIR -B $BUILD_ROOT
+    cmake --build $BUILD_ROOT -j 16 --target install
```

Comment:
should we copy install commands from readme directly?
currently we only need to build `cutlass_profiler`?

Reply:
The CMakefile.txt of cutlass has defined `install` target and allow user to specify where to install the binary and library, we can just use it directly. And it also has defined some switches to control what target to build, for now, only cutlass_profiler will be built. 
If we just use the `build` command from readme, we should add the `install` logic by ourselves.